ci/deps/kokkos-kernels.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+
+if test $# -lt 2
+then
+    echo "usage: ./$0 <kokkos-kernels-version> <build-dir>"
+    exit 1
+fi
+
+kokkos_version=$1
+kokkos_zip_name=${kokkos_version}.zip
+
+build_dir=$2
+
+echo "${kokkos_version}"
+echo "${kokkos_zip_name}"
+
+wget http://github.com/kokkos/kokkos-kernels/archive/${kokkos_zip_name}
```

Comment:
shellcheck warns:

> SC2086: Double quote to prevent globbing and word splitting.
Did you mean:
wget http://github.com/kokkos/kokkos-kernels/archive/"${kokkos_zip_name}"

```suggestion
wget http://github.com/kokkos/kokkos-kernels/archive/"${kokkos_zip_name}"
```

Reply:
It would probably be preferable to just quote the entire argument, in most of these cases, rather than just the variable expansions.

Reply:
I get why these checkers throw these warnings, but I do know that the `kokkos_zip_name` never contains a space. However, I will make these changes.

Reply:
Fixed